docs/cryptodoc/src/05_07_spx.rst

```diff
@@ -199,183 +207,211 @@ signing one.
+   +-------------------------+-----------+-----------+-----------+------------+-----------+-----------+--------------+-----------+
+   | ``SLH-DSA-<hash>-192s`` | 24        | 63        | 7         | 9          | 14        | 17        | 4            | 39        |
+   +-------------------------+-----------+-----------+-----------+------------+-----------+-----------+--------------+-----------+
+   | ``SLH-DSA-<hash>-192f`` | 24        | 66        | 22        | 3          | 8         | 33        | 4            | 42        |
+   +-------------------------+-----------+-----------+-----------+------------+-----------+-----------+--------------+-----------+
+   | ``SLH-DSA-<hash>-256s`` | 32        | 64        | 8         | 8          | 14        | 22        | 4            | 47        |
+   +-------------------------+-----------+-----------+-----------+------------+-----------+-----------+--------------+-----------+
+   | ``SLH-DSA-<hash>-256f`` | 32        | 68        | 17        | 4          | 9         | 35        | 4            | 49        |
+   +-------------------------+-----------+-----------+-----------+------------+-----------+-----------+--------------+-----------+
+
+SLH-DSA key generation follows Section 10.1 of [FIPS-205]_ and is
+implemented in :srcref:`[src/lib/pubkey/sphincsplus/sphincsplus_common]/sphincsplus.cpp:306|SphincsPlus_PrivateKey`
 within the ``SphincsPlus_PrivateKey`` constructor. It works as follows:
 
-.. admonition:: SPHINCS+ Key Generation
+.. admonition:: SLH-DSA Key Generation
 
    **Input:**
 
    -  ``rng``: random number generator
-   -  ``params``: SPHINCS\ :sup:`+` parameters
+   -  ``params``: SLH-DSA parameters
 
    **Output:**
 
    -  ``SK``, ``PK``: private and public key
 
    **Steps:**
 
-   1. Generate new values ``secret_seed``, ``prf``, and ``public_seed`` using ``rng``.
-   2. ``sphincs_root = xmss_gen_root(secret_seed)``
-      (see :ref:`SPHINCS+ XMSS <signatures/sphincsplus/XMSS>`).
-   3. | ``SK = {secret_seed, prf, public_seed, sphincs_root}``
-      | ``PK = {public_seed, sphincs_root}``
+   1. Generate new values ``sk_seed``, ``sk_prf``, and ``pub_seed`` using ``rng``.
+   2. ``root = xmss_gen_root(secret_seed)``
+      (see :ref:`XMSS <signatures/sphincsplus/XMSS>`).
+   3. | ``SK = {sk_seed, sk_prf, pub_seed, root}``
+      | ``PK = {pub_seed, root}``
 
    **Notes:**
 
    - The creation of a public key is conducted using the
      ``public_key`` method of the private key.
+   - The addresses are set according to Algorithm 18 of [FIPS-205]_.
+
+SPHINCS\ :sup:`+`
+^^^^^^^^^^^^^^^^^
+
+Botan supports the SPHINCS\ :sup:`+` Round 3.1 NIST submission [SPX-R3]_. The
+SPHINCS\ :sup:`+` instances are activated using the ``sphincsplus_sha2`` and
+``sphincsplus_shake`` modules, enabling their selection for key creation. These
+instances are maintained solely for version compatibility. It is strongly
```

Comment:
Maybe mention that these are available as SLH-DSA Parameter sets? To clarify it's not a separate implementation.